lib/iris/coords.py

```diff
@@ -2775,6 +2779,366 @@ def xml_element(self, doc):
         return cellMethod_xml_element
 
 
+class Connectivity(_DimensionalMetadata):
+    """
+    A CF-UGRID topology connectivity, describing the topological relationship
+    between two lists of dimensional elements. One or more connectivities
+    make up a CF-UGRID topology - a constituent of a CF-UGRID mesh.
```

Comment:
Not quite sure I agree on the description here...

The `Connectivity` is a definition (in terms of indices) of the interconnection between geometrical elements of a mesh, see [here](https://ugrid-conventions.github.io/ugrid-conventions/#topology).

Could we make this doc-string a wee bit more specific to the role and purpose of `Connectivity`?

Reply:
I really struggled to come up with a decent description, any help is appreciated.

Reply:
@bjlittle please could you provide a suggestion for us to talk over? Thanks 🍺